projboard/tests/test_client.py

```diff
@@ -1,7 +1,39 @@
 import pytest
+from projboard.models.article import Article
 
 
 @pytest.mark.django_db
 def test_homepage(client):
     response = client.get("/")
     assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_create_article_get(client):
+    response = client.get('/create_article/')
+    assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_create_article_post(client, article, django_capture_on_commit_callbacks):
+    assert len(Article.search_by_title(article.title + ' form')) == 0
```

Comment:
Why do you check it in DB? 
Please call a fixture that search and holds the results
this way, test need only client, and fixture will handle db

Reply:
When we create the page of my_articles I will test the results.
 I opened an Issue for that #75 
Done at [98a2d9a](https://github.com/redhat-beyond/Smarticle/pull/70/commits/98a2d9af404f43b381df51438456b221a003e754)